Update audit-log-events.md (#9732)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/audit-log-events.md

|Added workforce integration|WorkforceIntegrationAdded|The Shifts app is integrated with a third-party system.|
|Accepted off shift message|OffShiftDialogAccepted|A user acknowledges the off-shift message to access Teams after shift hours.|

## Updates app in Teams activities

> If your organization is using the Updates app in Teams, you can search the audit log for activities related to the Updates app. Here's a list of all events that are logged for Updates App activities in Teams in the Microsoft 365 audit log.

Reviewer: ```suggestion
If your organization is using the Updates app in Teams, you can search the audit log for activities related to the Updates app. Here's a list of all events that are logged for the Updates app activities in Teams in the Microsoft 365 audit log.
```

|Friendly name|Operation|Description|
|---|---|---|
|Create an update request|CreateUpdateRequest|A user successfully creates an update request.|

> |Edite an update request|EditUpdateRequest|A user opens request editing wizard and hits ‘save’ button to confirm and save any changes, or enable/disable the update request directly.|

Reviewer: ```suggestion
|Edite an update request|EditUpdateRequest|A user opens the request editing wizard and selects **Save** to confirm and save any changes or enable or disable the update request directly.|
```

> |Submit an update|SubmitUpdate|A user successfully submits an update..|

Reviewer: ```suggestion
|Submit an update|SubmitUpdate|A user successfully submits an update.|
```
